ground/gcs/src/libs/libcrashreporter-qt/breakpad/client/linux/handler/exception_handler.cc

```diff
@@ -1,4 +1,5 @@
 // Copyright (c) 2010 Google Inc.
+// Copyright (C) 2015 dRonin, http://dronin.org
 // All rights reserved.
 //
 // Redistribution and use in source and binary forms, with or without
```

Comment:
I think these files we imported, but didn't change.  Shouldn't have our copyright.